File: Rosu/asd/lab/java/ASDLab/src/main/java/ro/razvan/uniud/asd/lab/es4/Principale.java
package ro.razvan.uniud.asd.lab.es4;

import java.util.Arrays;
import java.util.Scanner;

class Principale {

    public static void main(String[] args) {
        Arrays.stream(StringUtils.prefixes(new Scanner(System.in).nextLine()))
                .forEach(System.out::println);
    }
}

File: Rosu/programmazione/laboratorio/java/esercitazioni/2019-03-29/ii/ones-complement/src/ro/razvan/uniud/onesComplement/Principale.kt
package ro.razvan.uniud.onesComplement

import ro.razvan.uniud.onesComplement.IterationStrings.onesComplement

fun main() {
    testing()
}

private fun testing() {
    println(onesComplement("1111") == "0000")
    println(onesComplement("101") == "010")
}